File: trail1/A/B 출력/a-divide-b.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        string[] arr = Console.ReadLine().Split();
        var(A, B) = (int.Parse(arr[0]), int.Parse(arr[1]));
        Console.Write(A/B + ".");

        int remainder = A % B;
        for (int i=0; i<20; i++) {
            remainder *= 10;
            Console.Write(remainder / B);
            remainder %= B;
        }
    }
}

File: trail1/A/B 출력/a-divide-b.cpp
#include <iostream>
using namespace std;

int main() {
    int A, B;
    cin >> A >> B;
    cout << A / B << ".";
    int remainder = A % B;
    for (int i=0; i<20; i++) {
        remainder *= 10;
        cout << remainder / B;
        remainder %= B;
    }
    return 0;
}